cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,13 +91,12 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
-    protected static final String CRUISE_CONTROL_CONFIG_VOLUME_NAME = "config";
-    protected static final String CRUISE_CONTROL_SERVER_CONFIG_FILENAME = "server-config";
-    protected static final String CRUISE_CONTROL_CAPACITY_CONFIG_FILENAME = "capacity-config";
+    protected static final String CONFIG_VOLUME_NAME = "config";
+    protected static final String SERVER_CONFIG_FILENAME = "server.properties";
```

Comment:
cruisecontrol.properties? there is no server.properties in CC config dir

Reply:
Thanks for the catch, fixed